guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -96,4 +96,7 @@ You have the option to return to any part of the job wizard and edit the informa
 ----
 +
 Replace `_My_Search_Query_` with the filter expression that defines hosts, for example `"name ~ _My_Pattern_"`.
-For more information about executing remote commands with hammer, enter `hammer job-template --help` and `hammer job-invocation --help`.
+
+.Additional resources
+
+* For more information about creating, monitoring, or canceling remote jobs with hammer, enter `hammer job-template --help` and `hammer job-invocation --help`.
```

Comment:
```suggestion
* For more information about creating, monitoring, or canceling remote jobs with Hammer CLI, enter `hammer job-template --help` and `hammer job-invocation --help`.
```

We capitalize "Hammer": https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization

CLI is probably optional.